content/guides/getstarted/rust-to-solana.md

```diff
@@ -0,0 +1,273 @@
+compute units with `sol_log_compute_units`.
+
+## Learning the Development Environment and Frameworks
+
+While the Solana CLI and the `solana_program` crate are all that is needed to
+get started, there are a couple of helpful tools which can accelerate learning.
+
+<div align="center">
+    <img src="./rust-to-solana-img/network-diagram.png" alt="Network Diagram" style="max-width: 80%; height: auto;"/>
+</div>
+
+### Solana Playground
+
+The [Solana Playground](https://beta.solpg.io/) is a browser-based IDE that
+allows developers to develop and deploy Solana programs.
+
+<div align="center">
+    <img src="./rust-to-solana-img/solana-playground.png" alt="Solana Playground" style="max-width: 80%; height: auto;"/>
+</div>
+
+It’s the easiest way to begin developing with Solana, and it supports building,
+testing, and deploying Solana Rust programs. Additionally, a number of built-in
+tutorials are available to guide learning.
+
+The Solana Playground allows developers to get started immediately without
+setting up a local environment.
+
+### Using Anchor
+
+[Anchor](https://www.anchor-lang.com/) is a framework that seeks to accelerate
+the building of secure Solana programs. It can help by handling standard
+boilerplate code, speeding up the development cycle. Additionally, it provides
+some security checks by default, making Solana programs more secure.
+
+To create a new program, simply create a new Anchor project in the Solana
+playground.
+
+Alternatively,
+[install the Anchor CLI](https://www.anchor-lang.com/docs/installation) locally,
+and then use `anchor init <project-name>` to create a new Anchor project.
+
+## Creating Off-chain Programs
+
+So far, this guide has covered the key details of developing **onchain Solana
+programs** in Rust. However, it’s also possible to develop **off-chain Solana
+clients** in Rust. This can be done by using the
+[solana_sdk crate](https://docs.rs/solana-sdk/latest/solana_sdk/). This contains
+the [solana_client crate](https://docs.rs/solana-client/latest/solana_client/)
+that allows Rust programs to interact with a Solana node via the
+[JSON RPC API](https://solana.com/docs/rpc).
+
+Another option is to use the
+[anchor_client crate](https://docs.rs/anchor-client/latest/anchor_client/) which
+interacts with Solana programs written in Anchor via RPC. Alternatively,
+consider writing onchain programs in Rust, and off-chain [clients in JS/TS](https://solana.com/de/docs/clients/javascript-reference).
+
+## Wrap Up
+
+This guide has covered the basics of developing for Solana with Rust, from setup
+details and restrictions to development environments and frameworks.
+
+For more Rust-related Solana resources, check out the
+[Developing with Rust page](https://solana.com/docs/programs/lang-rust). And for
+other Solana program examples written with Rust, check out these
+[examples on GitHub](https://github.com/solana-labs/solana-program-library/tree/master/examples/rust).
```

Comment:
Can you add a link to the program examples page in the docs and a sentence that makes sense for it:
https://solana.com/docs/programs/examples